scripts/e2e/generate-e2e-spec.sh

```diff
@@ -0,0 +1,24 @@
+echo "Creating e2e spec files using 'generate-yamls.sh'..."
```

Comment:
This file needs a license header on top of it.

Reply:
Done.